WORKSPACE

```diff
@@ -16,9 +16,25 @@ workspace(name = "io_opentelemetry_cpp")
 
 # Load our direct dependencies.
 load("//bazel:repository.bzl", "opentelemetry_cpp_deps")
+load("//bazel:openssl_repositories.bzl", "openssl_repositories")
 
 opentelemetry_cpp_deps()
 
+openssl_repositories()
```

Comment:
Is opessl a dependency for boost? I mean does thrift uses the features of boost which require openssl ( i know boost::asyncio requires openssl).

Reply:
It is needed for [thrift](https://github.com/esigo/opentelemetry-cpp/blob/867af702b2df856b79ed65f8d44f86c490984c88/exporters/jaeger/BUILD#L212).